### PR TITLE
Tweak settings menu

### LINKED ISSUE #2775: Settings subitems change depending on setting level
The (number of) items of the settings menu change depending on the setting level chosen from within one of the settings subitems. 
I think it's confusing that the items change at all and in particular that to change them one has first go into one of the setting subitems to change the setting level.
Basic
Screenshot at 2023-10-23 21-42-24

Advanced
Screenshot at 2023-10-23 21-42-39

Expert
Screenshot at 2023-10-23 21-42-56

### PR DESCRIPTION
**Pull request by yubiuser**
**What does this PR aim to accomplish?:**

1) Show all subitems of the settings menu in the sidebar regardless of the chosen setting level.
2) Ensure that at least one item is shown on each setting page (affects Local DNS Records & Advanced Settings)
3) Apply wrench and cog symbol to all settings.

---
**By submitting this pull request, I confirm the following:**

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against.
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
6. I have checked that another pull request for this purpose does not exist.
7. I have considered, and confirmed that this submission will be valuable to others.
8. I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
9. I give this submission freely, and claim no ownership to its content.

---
- [x] I have read the above and my PR is ready for review. *Check this box to confirm*

### CODE REVIEW

**Comment by yubiuser** [comment]
Ready for discussion/review

File: settings-dnsrecords.lp [hunk 1]
```diff
@@ -14,10 +14,10 @@ PageTitle = "Local DNS Settings"
 mg.include('scripts/pi-hole/lua/settings_header.lp','r')
 ?>
 <div class="row">
-    <div class="col-md-6 settings-level-1">
+    <div class="col-md-6 settings-level-0">
         <div class="box">
             <div class="box-header with-border">
-                <h3 class="box-title">Local DNS records</h3>
+                <h3 class="box-title">Local DNS records&nbsp;&nbsp;<i class="fas fa-wrench" title="This is an advanced-level setting"></i></h3>
```
Comment (rdwebdesign) [inline settings-dnsrecords.lp:20]: This doesn't look right.

I'm thinking about a solution, but I don't like the idea of changing the level to zero if the option is Advanced.

Comment (yubiuser) [reply settings-dnsrecords.lp:20]: I agree... We could make this a general basic level setting or move itto `DNS`.

**Comment by DL6ER** [comment]
I disagree with this PR and the some of the points brought up in the related issue #2775

(see comment over there)

**Comment by yubiuser** [comment]
Closing after internal discussion.

**Pull request closed without merging**